### DISCUSSION TITLE
Current Spring Boot version 3.0 and 3.2 not covered

### DISCUSSION THREAD

**Discussion by MuellerKlaus** [community/community#111442] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hi,

Copilot mention the last known release version of Spring boot as 2.6.x . It does know Spring Boot 3.0M1 only which is outdated.

When and how the latest version will be supported?

Response from Copilot:

> I'm sorry for any confusion, but as of my last training data, there is no Spring Boot version 3.2. The latest stable release of Spring Boot is 2.6.x, and the first milestone of Spring Boot 3.0.0 has been released recently.

**Comment by saifeddineelhanoune** [comment]
As of the latest information available, Spring Boot 3.2.x was released on November 23, 2023, and is expected to be supported until November 23, 2026, with commercial support ending on February 23, 2026

**Reply by MuellerKlaus** [reply]
That is not the question.

The problem is that Copilot does answer SB question with SB 2.6 code versions. There are some changes with WB 3.0/3.2. Copilot did not learn something for SB3.x
